CONTRIBUTING.md

```diff
@@ -436,11 +436,16 @@ is to be helpful, not to turn writing code into a chore.
        regular comments in the code. Remember as well that Elasticsearch
        has extensive [user documentation](./docs), and it is not the role
        of Javadoc to replace that.
+        1. If a method's performance is "unexpected" then its good to call that
+           out in the javadoc. Especially if its usually fast but sometimes
+           very slow (shakes fist at caching).
    12. Please still try to make class, method or variable names as
        descriptive and concise as possible, as opposed to relying solely on
        Javadoc to describe something.
-   13. Use `@link` and `@see` to add references, either to related
-       resources in the codebase or to relevant external resources.
+   13. Use `@link` to add references to related resources in the codebase. Or
+       outside the code base.
+       1. `@see` is much more limited than `@link`. You can use it but most of
+          the time `@link` flows better.
```

Comment:
There's a good discussion here - https://stackoverflow.com/questions/10097199/javadoc-see-or-link - worth linking to it?

Reply:
Yeah - we can. I come down on the other side from this answer though. I think if its important you should mention it in the prose. If you mention something twice you should link the first time and not again. Either `@linkplain` or don't link at all.